yellowbrick/cluster/elbow.py

```diff
@@ -262,6 +270,9 @@ def fit(self, X, y=None, **kwargs):
                 self.scoring_metric(X, self.estimator.labels_)
             )
 
+        self.kneedle=KneeLocator(self.k_values_,self.k_scores_,curve='convex',direction='decreasing')
```

Comment:
I'm a bit concerned about the `curve` and `direction` parameters of the `KneeLocator` - depending on the `metric` parameter, the curve may be concave/increasing or convex/decreasing (and other, unimplemented metrics may have a different effect). 

This leads me to believe that we need to do one of the following:

- Allow the user to specify the expected curve/direction
- Set the curve/direction based on the metric choice 
- Try multiple locators with different curve characteristics if we can't find an elbow 
- Normalize the curve to always be convex/decreasing no matter the metric
- Some combination of one or more of the above

Potentially the second bullet point is the easiest option, and we can go that route and open an issue to explore this further. What do you think? Are there any other options?

Reply:
@bbengfort I was also concerned about the same , it didn't feel right to me to pass these two parameters manually. Regarding first bullet point , I think it would not be convenient for the users (new to ML) to specify the expected curve and direction. I think for now 2nd point is the most viable option given we have only three metrics for now. And if new metrics are to be included in future we can modify the same. I'll update this soon. Thanks.